### PR TITLE
Fix multiple factors issue

### PR DESCRIPTION
**Pull request by kbodwin**
`fct_cross` was breaking with more than one additional factor, because the default variable name for `tibble::tibble()` is "<fct>", and so there were duplicate variable names.  I've added the option `.name_repair` to take care of this, and suppressed the resulting message.

Addresses #182

### CODE REVIEW

File: R/fct_cross.R [hunk 1]
```diff
@@ -24,7 +24,8 @@ fct_cross <- function(.f, ..., sep = ":", keep_empty = FALSE) {
     return(.f)
   }
 
-  .data <- lapply(tibble::tibble(.f, !!!flist), check_factor)
+  .data <- suppressMessages(tibble::tibble(.f, !!!flist, .name_repair = "universal"))
```
Comment (hadley) [inline R/fct_cross.R]: Hmmmmm, why does it get put into a tibble in the first place? I don't think the names are ever actually used?

**Comment by hadley** [comment]
(BTW you need to use specifically `Fixes #182` or `Closes #182` to activate the github magic that will cause the issue to be closed when the PR is merged)

File: R/fct_cross.R (hunk 1 above)
Comment (kbodwin) [reply R/fct_cross.R]: I'm not totally sure, but I think it's just a convenience for getting all the factors together when the length of `flist` is unknown.

I actually think it makes the most sense to just not have `.f` as a separate input...

**Comment by kbodwin** [comment]
> (BTW you need to use specifically `Fixes #182` or `Closes #182` to activate the github magic that will cause the issue to be closed when the PR is merged)

Thank you!  This should be in the PR description or in the commit message or both?

Regardless, I believe the above commit Fixes #182

**Comment by hadley** [comment]
Would you mind also providing a unit test that checks that crossing 2+ factors now works?

File: R/fct_cross.R (hunk 1 above)
Comment (hadley) [reply R/fct_cross.R]: It was actually needed for recycling, so I've bought it back with a test.

**Comment by kbodwin** [comment]
Incorporated changes into #236

**Pull request closed without merging**